runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientDescribeFactory.java

```diff
@@ -821,7 +824,7 @@ private void doApplicationAbort(
             state = KafkaState.closedReply(state);
             //client.stream = nullIfClosed(state, client.stream);
             doAbort(application, originId, routedId, replyId, replySeq, replyAck, replyMax,
-                    traceId, client.authorization, EMPTY_EXTENSION);
+                traceId, client.authorization, EMPTY_EXTENSION);
```

Comment:
Do we need to make it conditional as well so that we don't send extra unnecessary frame

Reply:
Yes, agreed - it is already conditional in `doApplicationAbortIfNecessary` though.